examples/2-query/src/GetAll.re

```diff
@@ -0,0 +1,89 @@
+/* Opens our styles module */
+open GetAllStyles;
+
+type state = {
+  listOfPokemons: Js.Array.t(string),
+  textInput: string,
+  filteredList: Js.Array.t(string),
+  selectedPokemon: Js.Nullable.t(string),
```

Comment:
How would you feel about making this an `option(string)` type?

Reply:
Done, much nicer!